GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/MountTests.cs

```diff
@@ -213,16 +226,27 @@ public void MountRegeneratesAlternatesFileWhenMissingFromDisk()
         [TestCase]
         public void MountCanProcessSavedBackgroundQueueTasks()
         {
-            this.Enlistment.UnmountGVFS();
-
             string deleteFilePath = Path.Combine("Test_EPF_WorkingDirectoryTests", "1", "2", "3", "4", "ReadDeepProjectedFile.cpp");
             string deleteDirPath = Path.Combine("Test_EPF_WorkingDirectoryTests", "1", "2", "3", "4");

```

Comment:
Minor style comment:  You could move `deleteFilePath` and `deleteDirPath` down below this comment:

`// Prime the background queue with delete messages`

That might make the relationship between `deletedFileEntry`/`deletedDirEntry ` and ModifiedPaths.dat more clear.

Reply:
Sure.  I did see value in keeping the vars together too.  Aka, this is the file definition this is the entry.
I have moved it are per your suggestion, as it's the type of thing that could go either way (at least in my mind).

Reply:
Ahh I can see that.  I think for me what threw me off a bit was whether "deletedFileEntry" meant an entry in the background tasks file or an entry in the modified paths file.  Either way is good with me if you prefer the original.